src/BookFormModal.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+import React from 'react';
+import Modal from 'react-bootstrap/Modal'
+import Button from 'react-bootstrap/Button'
+import Form from 'react-bootstrap/Form'
+// import Form from 'react-bootstrap/Form';
+
+class BookFormModal extends React.Component {
+  render() {
+
+    return (
+      <>
+        <Button variant="primary" onClick={this.props.handleShowModal}>
+          Launch demo modal
+        </Button>
```

Comment:
- Do we need this button inside this component? I believe all we need is the Modal and Form below 👍 
- This button would open modal, but it should live outside the Modal component. Then when the user clicks on it, it shows this modal :)